Revisit searcher docs (#165)
api3dao/api3-docs: docs/oev-searchers/in-depth/oev-network/index.md

| Block Explorer URL | https://oev.explorer.api3.org/ |
| Bridge URL         | https://oev.bridge.api3.org/   |

::: info 💡 Tip

> Apart from the official bridge, Api3 hosts an extended version available on https://oev-explorer-extended.api3.org/. This is a fork of the explorer with a small additions for relevant for OEV.

Reviewer: I wasn't aware of this 🤯 

Nit:

```suggestion
Apart from the official bridge, API3 hosts an extended version available at https://oev-explorer-extended.api3.org/. This is a fork of the explorer with small additions relevant to OEV.
```
